com.sublimetext.three.json

```diff
@@ -75,13 +91,16 @@
                 {
                     "type": "file",
                     "path": "sublime.sh"
-                },
-                {
-                    "type": "file",
-                    "path": "sublime-Makefile",
-                    "dest-filename": "Makefile"
                 }
             ],
+            "build-commands": [
+                "mkdir -p /app/bin",
+                "mkdir -p /app/lib",
+                "install apply_extra /app/bin",
```

Comment:
over at flathub we prefer [single line install commands](https://github.com/flathub/io.atom.Atom/blob/master/io.atom.Atom.json#L46) for single files like this, which gets rid of the mkdir as well. Not a blocker by any means.